lib/single-new-eks-opensource-observability-construct/rules-1.ytpl

```diff
@@ -0,0 +1,214 @@
+apiVersion: prometheusservice.services.k8s.aws/v1alpha1
```

Comment:
These are lengthy Yamls. Do we have an opprtunity to use GitOps to deploy this to EKS cluster vs deploying it via cluster addon. We have a problem of maintenance in static and with GitOps its better way of managing kubernetes resoruces?